### ISSUE TITLE
Pronunciation guides

### ISSUE THREAD

**Issue by TazeTSchnitzel** [witch-house/pronoun.is#43]
Since English's spelling is notoriously inconsistent, it might be helpful to give people a guide to pronunciation for some of these. This would be especially helpful for non-native English speakers, who are probably more likely to make a poor guess of pronunciation.

A given pronoun set might have multiple conceivable pronunciations. I don't think we could really say which is more correct, we can list whichever we're aware of. We'd have to crowdsource these much like we do pronoun sets themselves.

The guides could consist of, for example, International Phonetic Alphabet (e.g. “zir” might be UK: /ˈzɜː(ɹ)/, US: /ˈzɝ/ using [Wiktionary's English IPA key](https://en.wiktionary.org/wiki/Appendix:English_pronunciation)), re-spelled English (e.g. “zir” might be “zər” using [Wikipedia's respelling key](https://en.wikipedia.org/wiki/Help:Pronunciation_respelling_key)), and an intuitive guide (e.g. ‘”zir” is pronounced like “her” but with the “h” replaced with a “z”’).

Thoughts?

**Comment by morganastra** [comment]
I agree this would be a great feature to have, but there's not an obvious way to implement this right now.

I just added my thoughts to the FAQ also https://github.com/witch-house/pronoun.is#can-we-add-pronunciation-guides

**Comment by ghost** [comment]
Trying to support multiple pronunciation schemes (which is probably necessary) will make it hard to keep the defaults table simple and intuitive, but a starting point may be to add support via query parameters. Maybe something like `pronoun.is/he/him?pronounced=HEE/HIMM&pronounced=IPA:ˈhiː/ˈhɪm`

For the defaults, maybe the best solution is to add separate tables (getting a little bit SQL-y). The first column could identify the pronoun set using the same conventions as the resource paths:
```
he                   HEE     HIHM     HIHZ     HIHZ      HIHM-SEHLF
they/.../themself    THAY    THEHM    THEHR    THEHRS    THEHM-SEHLF
```
Separate files can be used to define whatever extra phonetic spelling schemes we'd want to support (ipa.tab, enpr.tab, etc.).

**Comment by b1f6c1c4** [comment]
A straightforward way for doing this is something like

https://pronoun.is/she%20%5B%CA%83i%5D/her%20%5Bh%C9%9D%5D/her%20%5Bh%C9%9D%5D/hers%20%5Bh%C9%9Dz%CC%A5%5D/herself%20%5Bh%C9%9D%CB%88s%C9%9B%C9%ABf%5D

And there _may_ be a redirection from https://pronoun.is/she?p=IPA&d=GenAm to the link above.

Any idea on this issue? @untunt
